lookout/style/format/features.py

```diff
@@ -146,12 +153,15 @@ def to_comment(self, correct_y: int) -> Comment:
 CLASSES = (CLS_SPACE, CLS_TAB, CLS_NEWLINE, CLS_SPACE_INC, CLS_SPACE_DEC,
            CLS_TAB_INC, CLS_TAB_DEC, CLS_SINGLE_QUOTE, CLS_DOUBLE_QUOTE, CLS_NOOP)
 CLASS_INDEX = {cls: i for i, cls in enumerate(CLASSES)}
+EMPTY_CLS = frozenset([CLASS_INDEX[CLS_TAB_DEC], CLASS_INDEX[CLS_SPACE_DEC],
```

Comment:
why do you need to extract hash value from `EMPTY_CLS`? Or you want to make this structure unmodifiable?

Reply:
it's a reflex: when I use something primarily for `in` operations I use sets. And this won't be modified so `frozenset` in this case.